CONTRIBUTIONS.md

```diff
@@ -0,0 +1,46 @@
+# ![Moonbeam](media/moonbeam-cover.jpg)
+
+## Contributions
+
+Moonbeam primarily uses GitHub Pull Requests to orchestrate code changes. If you wish to propose a
```

Comment:
```suggestion
Moonbeam primarily uses GitHub Pull Requests to coordinate code changes. If you wish to propose a
```
Just a personal preference, feel free to ignore

Reply:
Lol, fair enough